README.md

```diff
@@ -1,3 +1,6 @@
+# About
 This repo holds some of the sources for the Permissions API, an interface for Web applications to be able to manage permissions.
 
 The latest editor’s draft is at https://w3c.github.io/permissions/
+
+# [Contributing](https://github.com/w3c/permissions/blob/main/.github/CONTRIBUTING.md)
```

Comment:
Adding a heading is a great idea. I like the way this repo does it: https://github.com/webcompat/webcompat.com#contributing

Perhaps rather than linking the heading we can write a simple sentence like:

"We welcome contributions to this repo. For more info, please see [CONTRIBUTING]( link here )."

Reply:
How does it look now? Is there more information I should add to it?